src/System.Security.Claims/src/System/Security/Claims/Claim.cs

```diff
@@ -548,7 +543,7 @@ protected virtual void WriteTo(BinaryWriter writer, byte[] userData)
         /// <returns>The string representation of the <see cref="Claim"/> object.</returns>
         public override string ToString()
         {
-            return String.Format(System.Globalization.CultureInfo.InvariantCulture, "{0}: {1}", _type, _value);
+            return _type + ": " + _value;
```

Comment:
Seems like the original InvariantCulture is more correct for literal "ToString" methods; I believe CurrentCulture is the default. netfx uses InvariantCulture